test/js-unit/recrasContactFormSpec.js

```diff
@@ -219,6 +219,75 @@ describe('RecrasContactForm', () => {
                 expect(sorted[1]).toEqual(packs[0]);
                 expect(sorted[2]).toEqual(packs[1]);
             });
-        })
+        });
+
+        describe('getRequiredFields', () => {
+            let rc;
+
+            beforeEach(async () => {
+                rc = new RecrasContactForm(new RecrasOptions({
+                    element: this.mainEl,
+                    form_id: 1,
+                    recras_hostname: 'demo.recras.nl',
```

Comment:
are these tests actually consulting `demo.recras.nl`? That would mean the tests break depending if the test runner has no or limited internet access or the data at `demo.recras.nl` changes.

Reply:
No, so we could change this to something else/

Reply:
I've created a separate issue to address this: #42